site/bulletin_board/data/users.js

```diff
@@ -126,4 +130,34 @@ users.get = (id, callback) => {
   });
 };
 
+
+users.retrieve = (id, userId, callback) => {
```

Comment:
I know we discussed "in person", but just a reminder that this is the same as `get`.